server/portal/apps/tickets/utils.py

```diff
@@ -0,0 +1,14 @@
+from django.conf import settings
+import requests
+
+
+def get_recaptcha_verification(request):
```

Comment:
Did we have a unit test for this new function?

Reply:
@rstijerina 
I thought `test_ticket_create_authenticated` and `test_ticket_create_authenticated_setup_incomplete` is sufficient but I could add a unit test specific for this function

Reply:
@williamkwon97 Yeah, can you add some test cases? Neither of those existing tests are `unauthenticated` test cases, so don't test this code.